### PR TITLE
polychromatic: fix qt5 wrapper to contain all required libraries

### LINKED ISSUE #200780: Trying to launch polychomatic getting error [qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""]
### Steps To Reproduce
Install base system and enable openrazer, install polychromatic from unstable
launch polychromatic-controller in terminal

### Expected behavior
polychromatic launches

### Notify maintainers
@SuperSandro2000 @evanjs 

### Metadata

```console
- system: `"x86_64-linux"`
 - host os: `Linux 5.15.77, NixOS, 22.11 (Raccoon), 22.11.20221109.872fcee`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(jrgiacone): `""`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by jrgiacone**
###### Description of changes
Fixes the error preventing polychromatic from launching.
Fixes https://github.com/NixOS/nixpkgs/issues/200780

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jrgiacone** [comment]
@evanjs @SuperSandro2000 I am not great with github so i rebased my branch and started over, this should fix the libxcb error, currently works on sway and hyprland with waybar, not working on xmonad with xmobar

**Comment by SuperSamus** [comment]
You need to squash the commits, and rename the commit message (and the PR title) to something like:
`polychromatic: fix libxcb and icon on waybar`
Use the commit description to tell what's still not working, not the message.
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

File: pkgs/applications/misc/polychromatic/default.nix [hunk 1]
```diff
@@ -41,32 +43,39 @@ python3Packages.buildPythonApplication rec {
...
 
+  nativeBuildInputs = with python3Packages; [
+    gettext
+    gobject-introspection
+    meson
+    ninja
+    sassc
+    wrapGAppsHook
+    qt5.wrapQtAppsHook
+  ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/polychromatic/default.nix:69]: Please move this back to its original position

File: pkgs/applications/misc/polychromatic/default.nix [hunk 2]
```diff
@@ -41,32 +43,39 @@ python3Packages.buildPythonApplication rec {
     scripts/build-styles.sh
   '';
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/polychromatic/default.nix:45]:
```suggestion
  nativeBuildInputs = with python3Packages; [
    gettext
    gobject-introspection
    meson
    ninja
    sassc
    wrapGAppsHook
    qt5.wrapQtAppsHook
  ];

```

**Comment by SuperSandro2000** [comment]
@ofborg build polychromatic

**Pull request merged**